src/php/bin/generate_proto_php.sh

```diff
@@ -28,13 +28,32 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-
 set +e
-cd $(dirname $0)
+cd $(dirname $0)/../../..
+
+protoc --proto_path=src/proto/math \
+       --php_out=src/php/tests/generated_code \
+       --grpc_out=src/php/tests/generated_code \
+       --plugin=protoc-gen-grpc=$(which grpc_php_plugin) \
+       src/proto/math/math.proto
 
-gen_code='../tests/generated_code'
-interop='../tests/interop'
+# replace the Empty message with EmptyMessage
+# because Empty is a PHP reserved word
```

Comment:
Actually at some point the protoc for PHP should be clever enough and have a list of reserved words and alter the names in the generated code so they don't collide. (Empty becomes `Empty_` or so). Is there a plan for that?